packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -99,7 +99,7 @@ export const RichTextBlock = memo(function RichTextBlock({
   return (
     <Slate
       editor={editor}
-      value={value}
+      value={value?.length ? value : WepublishEditor.createDefaultValue()}
```

Comment:
Allows to not having to set the correct default value as an empty array would have broken Slate.

This would fix potential bugs with the commentList as that for example set a default of empty array